Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [cetmix-tower-14.0-dev/cetmix_tower_server](https://hosted.weblate.org/projects/tower-server-14-0-dev/cetmix_tower_server/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/tower-server-14-0-dev/cetmix_tower_server/horizontal-auto.svg)

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->

## Summary by CodeRabbit

- **Bug Fixes**
  - Enhanced and refined Italian translations for improved clarity and consistency across the interface.
  - Corrected several textual inaccuracies in labels and messages to provide users with clearer guidance.

<!-- end of auto-generated comment: release notes by coderabbit.ai -->

Commit:
Added translation using Weblate (German)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/cetmix/cetmix-tower?pullRequest=207) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/cetmix/cetmix-tower?pullRequest=207) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: mymage<br/>:x: AquaMCU<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/cetmix/cetmix-tower?pullRequest=207) it.</sub>

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (483 of 483 strings)

Translation: cetmix-tower-14.0-dev/cetmix_tower_server
Translate-URL: https://hosted.weblate.org/projects/tower-server-14-0-dev/cetmix_tower_server/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (80 of 80 strings)

Translation: cetmix-tower-14.0-dev/cetmix_tower_yaml
Translate-URL: https://hosted.weblate.org/projects/tower-server-14-0-dev/cetmix_tower_yaml/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (85 of 85 strings)

Translation: cetmix-tower-14.0-dev/cetmix_tower_git
Translate-URL: https://hosted.weblate.org/projects/tower-server-14-0-dev/cetmix_tower_git/it/

AquaMCU committed:
Translated using Weblate (German)

Currently translated at 4.1% (21 of 508 strings)

Translation: cetmix-tower-14.0-dev/cetmix_tower_server
Translate-URL: https://hosted.weblate.org/projects/tower-server-14-0-dev/cetmix_tower_server/de/

ivs-cetmix commented:
/ocabot merge nobump

CetmixGitDrone commented:
This PR looks fantastic, let's merge it!
Prepared branch [14.0-dev-ocabot-merge-pr-207-by-ivs-cetmix-bump-nobump](https://github.com/cetmix/cetmix-tower/commits/14.0-dev-ocabot-merge-pr-207-by-ivs-cetmix-bump-nobump), awaiting test results.

CetmixGitDrone commented:
Congratulations, your PR was merged at 9759ddb09e05b6462bbd6dbb8d4c2f766f573fd6. Thanks a lot for contributing to cetmix. ❤️
